test/test_facade/test_smac_facade.py

```diff
@@ -81,6 +91,36 @@ def test_pass_tae_runner_objective(self):
             scenario=self.scenario,
         )
 
+    def test_construct_runhistory2epm(self):
+        """Check default setup up for consistency"""
+        smbo = SMAC4AC(self.scenario)
+        self.assertTrue(type(smbo.solver.epm_chooser.rh2EPM) == RunHistory2EPM4Cost)
+        self.assertSetEqual(set(smbo.solver.epm_chooser.rh2EPM.success_states),
+                            {StatusType.SUCCESS, StatusType.CRASHED, StatusType.MEMOUT})
+        self.assertSetEqual(set(smbo.solver.epm_chooser.rh2EPM.impute_state), set())
+        self.assertSetEqual(set(smbo.solver.epm_chooser.rh2EPM.consider_for_higher_budgets_state),
+                            set())
+
+        for intensifier in (SuccessiveHalving, Hyperband):
+            smbo = SMAC4AC(self.scenario, intensifier=intensifier,
+                           intensifier_kwargs=self.sh_intensifier_kwargs)
+            self.assertTrue(type(smbo.solver.epm_chooser.rh2EPM) == RunHistory2EPM4Cost)
+            self.assertSetEqual(set(smbo.solver.epm_chooser.rh2EPM.success_states),
+                                {StatusType.SUCCESS, StatusType.CRASHED, StatusType.MEMOUT})
```

Comment:
Maybe you also want to check for the states to advance?

Reply:
What do you mean? Isn't line 111 doing this?